systems/node/notify/internal/server/notify.go

```diff
@@ -119,7 +119,7 @@ func (n *NotifyServer) Delete(ctx context.Context, req *pb.GetRequest) (*pb.Dele
 		return nil, err
 	}
 
-	route := n.baseRoutingKey.SetAction("delete").SetObject("notification").MustBuild()
+	route := n.baseRoutingKey.SetAction("delete").SetObject("node").MustBuild()
```

Comment:
We are deleting a **notification**, not a "node"

Reply:
We discussed last time and agree to change that to node as the key will have alreay notify which means we don't need notification again.node.notify.node.delete which is fine.

Reply:
Routing keys updated accordingly  @delvatt .